src/index.jsx

```diff
@@ -101,7 +102,6 @@ class ReactAudioPlayer extends Component {
         ref={(ref) => { this.audioEl = ref; }}
         onPlay={this.onPlay}
         loop={this.props.loop}
-        {...this.props}
```

Comment:
Also now that I think about it, this line is a bigger problem.  It was added in PR https://github.com/justinmc/react-audio-player/pull/26 by someone who is actually using this to add extra attributes to the audio tag that aren't explicitly supported by this project.  If we remove it now, it will break their use case.

However, the warning that you noticed is still a problem we need to fix.  I think I will attempt a solution in another PR.  I think what I really need to do is explicitly support every audio tag attribute and at that point I can remove the extra props spread here.